src/dashboard/style.scss

```diff
@@ -1,5 +1,7 @@
 @import "../blocks/plugins/feedback/editor.scss";
 
+@import url('https://fonts.cdnfonts.com/css/barlow');
```

Comment:
I agree with Cristi here, we can remove this and see if we can try for a similar style without loading an extra font.